src/__mocked__/apiSchema.d.ts

```diff
@@ -3,9 +3,15 @@ declare namespace MockComponents {
 
     export type Case = {
       id: number
+      teamTitle: string
```

Comment:
Think `team: { id: number, title: string }` will be more close to eventual API data structure

Reply:
True, fixed it